pkg/accountmanager/poller_test.go

```diff
@@ -0,0 +1,379 @@
+			ces                       *v1alpha1.CloudEntitySelector
+			accountPollerObj          *accountPoller
+			pollIntv                  uint
+			mockCtrl                  *mock.Controller
+			mockCloudInterface        *cloud.MockCloudInterface
+		)
+		BeforeEach(func() {
+			newScheme := runtime.NewScheme()
+			utilruntime.Must(clientgoscheme.AddToScheme(newScheme))
+			utilruntime.Must(v1alpha1.AddToScheme(newScheme))
+
+			fakeClient = fake.NewClientBuilder().WithScheme(newScheme).Build()
+			mockCtrl = mock.NewController(GinkgoT())
+			mockCloudInterface = cloud.NewMockCloudInterface(mockCtrl)
+			cloudInventory := inventory.InitInventory()
+			accountPollerObj = &accountPoller{
+				log:       logf.Log,
+				Client:    fakeClient,
+				inventory: cloudInventory,
+				mutex:     sync.RWMutex{},
+			}
+
+			pollIntv = 1
+			account = &v1alpha1.CloudProviderAccount{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      testAccountNamespacedName.Name,
+					Namespace: testAccountNamespacedName.Namespace,
+				},
+				Spec: v1alpha1.CloudProviderAccountSpec{
+					PollIntervalInSeconds: &pollIntv,
+					AWSConfig: &v1alpha1.CloudProviderAccountAWSConfig{
+						Region: []string{"us-east-1"},
+						SecretRef: &v1alpha1.SecretReference{
+							Name:      testSecretNamespacedName.Name,
+							Namespace: testSecretNamespacedName.Namespace,
+							Key:       credentials,
+						},
+					},
+				},
+			}
+			credential := `{"accessKeyId": "keyId","accessKeySecret": "keySecret"}`
+			secret = &corev1.Secret{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      testSecretNamespacedName.Name,
+					Namespace: testSecretNamespacedName.Namespace,
+				},
+				Data: map[string][]byte{
+					"credentials": []byte(credential),
+				},
+			}
+			ces = &v1alpha1.CloudEntitySelector{
+				ObjectMeta: v1.ObjectMeta{
+					Name:      testCesNamespacedName.Name,
+					Namespace: testCesNamespacedName.Namespace,
+					OwnerReferences: []v1.OwnerReference{
+						{
+							APIVersion: "crd.cloud.antrea.io/crdv1alpha1",
+							Kind:       "CloudProviderAccount",
+							Name:       testAccountNamespacedName.Name,
+						},
+					},
+				},
+				Spec: v1alpha1.CloudEntitySelectorSpec{
+					AccountName: testAccountNamespacedName.Name,
+					VMSelector: []v1alpha1.VirtualMachineSelector{
+						{
+							VpcMatch: &v1alpha1.EntityMatch{
+								MatchID: "xyz",
+							},
+							VMMatch: []v1alpha1.EntityMatch{
+								{
+									MatchID:   "xyz",
+									MatchName: "xyz",
+								},
+							},
+						},
+					},
+				},
+			}
+			accountPollerObj.initVmSelectorCache()
+			accountPollerObj.cloudInterface = mockCloudInterface
+		})
+		It("Add or update selector", func() {
+			_ = fakeClient.Create(context.Background(), secret)
+			_ = fakeClient.Create(context.Background(), account)
+
+			//Add selector.
+			accountPollerObj.addOrUpdateSelector(ces)
+			Expect(len(accountPollerObj.vmSelector.List())).To(Equal(1))
+
+			//Update selector using same config.
```

Comment:
Please make sure to add any comments this way.  I still see lot of comments do not follow this format.
`// Test comment.`

Reply:
Solved.